src/components/OrdersWidget/index.tsx

```diff
@@ -153,6 +153,50 @@ const OrdersWidget: React.FC = () => {
               </div>
             )}
           </div>
+
+          {pendingShownOrdersCount ? (
+            <div>
+              <div className="ordersContainer">
+                <CardTable
+                  $columns="minmax(5rem, min-content) minmax(13.625rem, 1fr) repeat(2, minmax(6.2rem, 0.6fr)) minmax(5.5rem, 0.6fr)"
+                  $cellSeparation="0.2rem"
+                  $rowSeparation="0.6rem"
+                >
+                  <thead>
+                    <tr>
+                      <th className="checked">
+                        <input
+                          type="checkbox"
+                          onChange={toggleSelectAll}
+                          checked={orders.length === markedForDeletion.size}
+                          disabled={deleting}
+                        />
```

Comment:
I think the checkbox should either be disabled or not present for pending orders, because we can't cancel.
Also, when checking the box it select only active orders.

Yes, this will be replaced soon, so might not be that relevant.

Reply:
yeah thats what I was thinking, leaving out any design stuff